doc/releases/pacific.rst

```diff
@@ -2,6 +2,239 @@
 Pacific
 =======
 
+v16.2.15 Pacific
+================
+
+This is the fifteenth backport release in the Pacific series.
+
+Notable Changes
+---------------
+
+* `ceph config dump --format <json|xml>` output will display the localized
+  option names instead of its normalized version. For e.g.,
```

Comment:
1. `s/its/their/ `, i.e.: `[...] will display the localized option names instead of their normalized versions`
1. `For e.g.` should be either `For example` or `E.g.` but not a mix of those.

Reply:
This suggestion makes sense.

Reply:
@yuriw this is a grammatical change under Notable Changes. Let's include this.